internal/test/server.go

```diff
@@ -99,8 +89,8 @@ func (s *server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func (s *server) GetManifest() []byte {
-	data, err := json.Marshal(s.manifest)
+func (s *server) GetVersions() []byte {
```

Comment:
be more consistent 😄 
```suggestion
func (s *server) GetEnvoyVersions() []byte {
```

Reply:
and no need to be exported